src/Promitor.Core.Scraping/Configuration/Serialization/v1/Core/V1Deserializer.cs

```diff
@@ -20,6 +20,8 @@ public class V1Deserializer : Deserializer<MetricsDeclarationV1>
             Map(definition => definition.Version)
                 .IsRequired()
                 .MapUsing(GetVersion);
+            Map(definition => definition.UseAzureMonitorSdk)
+                .WithDefault(false);
```

Comment:
Let's migrate by default, this should in theory even be technical implementation detail
```suggestion
                .WithDefault(true);
```